### ISSUE TITLE
Problems exporting screenfetch to online image hosters when using -u option

### ISSUE THREAD

**Issue by lotsofluna** [KittyKatt/screenFetch#314]
When using the -u option with screenfetch, some (possibly all) of the image hosters seem to be "broken, in that no link is provided, and possibly no data is uploaded to their servers.

Those tested that don't work, confirmed:
screenfetch -u imgur, teknik, pomf (really dead)

The others haven't been tested by me. I'd say this is worth looking into, due to possible changes in API structure or general site structure.

**Comment by jhelfman** [comment]
> When using the -u option with screenfetch, some (possibly all) of the image hosters seem to be "broken", in that no link is provided, and possibly no data is uploaded to their servers.
> 
> Those tested that don't work, confirmed:
> screenfetch -u imgur, teknik, pomf (really dead)
> 
> The others haven't been tested by me. I'd say this is worth looking into, due to possible changes in API structure or general site structure.

yes. validated and more information I found in bug #278  This can probably be closed after your comment is merged with this bug.

**Comment by KittyKatt** [comment]
I'll probably work on getting this working this week by removing pomf and getting the proper URL/API calls for teknik and imgur. Any other suggestions for popularly used image hosts will also be accepted.
